src/slice_transform.rs

```diff
@@ -41,15 +39,15 @@ impl SliceTransform {
         transform_fn: TransformFn,
         in_domain_fn: Option<InDomainFn>,
     ) -> SliceTransform {
-        let cb = Box::new(TransformCallback {
+        let cb = Box::into_raw(Box::new(TransformCallback {
             name: CString::new(name.as_bytes()).unwrap(),
```

Comment:
I'd propose to handle the `CString` creation. IMO, 'unwrap()` is a bad idea to use here.

Reply:
I agree, but I'm not sure if making this even more of a breaking change is worth it from a pragmatic perspective. What do you think?

Reply:
@ekmartin you mean if we add a handle then we should change a return type to `Option<SliseTransform>` or something like this ?